build.gradle

```diff
@@ -1,24 +1,26 @@
-project.ext {
-    SPINE_VERSION = '0.9.21-SNAPSHOT'
-    GAE_JAVA_VERSION = "0.9.21-SNAPSHOT"
-    PROTOBUF_VERSION = '3.2.0'
-    SLf4J_VERSION = "1.7.21"
-    PROTOBUF_DEPENDENCY = "com.google.protobuf:protoc:${project.PROTOBUF_VERSION}"
-    MAVEN_REPOSITORY_URL = 'http://maven.teamdev.com/repository/spine'
-    MAVEN_SNAPSHOT_REPOSITORY_URL = 'http://maven.teamdev.com/repository/spine-snapshots'
-
-    GUAVA_VERSION = "20.0"
-
-    IS_WINDOWS = org.gradle.internal.os.OperatingSystem.current().isWindows()
-
+ext {
     publishPlugin = "$rootDir/script/publish.gradle"
     credentialsPropertyFile = 'credentials.properties'
     projectsToPublish = ['datastore']
+
+    /**
+     * A map of the Spine project IDs to their versions.
+     */
+    spineDependencies = ['spine-server': VERSION,
+                         'spine-client': VERSION,
+                         'spine-core': VERSION,
```

Comment:
I think the `spine-core` and `spine-base` dependencies should not be here, as you'll get those anyway.